cerbero/utils/__init__.py

```diff
@@ -104,6 +104,8 @@ def to_winepath(path):
         path = 'z:\\%s' % path
         return path
 
+def get_wix_wine_prefix():
+    return '/tmp/flu_ccwp'
```

Comment:
Use a random generated path, several build in this same machine could be using the same path

Reply:
Or I could remove the removal part seen that its only a symlink unique to our buildsystem...

Reply:
I don't think that will work, you will still have the same issue of several build configuration using the same directory. Temporal directories should never be hardcoded when possible

Reply:
It is not a temporal directory but a symbolic link to the build directory. I located it in /tmp for this reason to be remove by the linux system at least when we reboot the machine. Concerning the "hack", yes it is a hack to workaround the wine/wix limitation. I guess this bug is not happening on windows, or the use of Z folder is hiding it.

Reply:
It's a temporary directory that is created and destroyed in the package step, it can't be hardcoded.
If you need a link for shorter paths that is persistent than create it your self, don't do it with cerbero, just link /srv/yourbuild to /the/real/long/path/.

Reply:
this is a limitation due to wix using .NET4 with wine. I think we should let cerbero perform this, seen that it is in /tmp/